src/app/system/external-services/amazon-s3/amazon-s3.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+    <button mat-raised-button color="primary" [routerLink]="['edit']">
+      <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+      Edit
+    </button>
+  </div>
+
+<div class="container">
+
+  <div class="mat-container-z8">
```

Comment:
The class is `mat-elevation-z8`. Make changes in all the files necessary. Also, a single div can be used here.